pywavefront/parser.py

```diff
@@ -63,5 +63,11 @@ def parse(self, line, dir):
                 args[i] = arg
             i += 1
 
-        parse_function = getattr(self, 'parse_%s' % line_type)
-        parse_function(args)
+        attrib = 'parse_%s' % line_type
+
+        if hasattr(self, attrib):
+            parse_function = getattr(self, attrib)
+            parse_function(args)
+        else:
+            print("[PyWavefront] WARNING: Unimplemented OBJ format statement \'%s\' on line \'%s\'"
```

Comment:
Please use `logging.warning` here, and remove the `[PyWavefront] WARNING` prefix.

The quotes in the output string do not need escaping IIRC. That is: `\'%s\'` should be safe to write as `'%s'`.